java/pattern/test/BUILD

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (C) 2020 Grakn Labs
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+package(default_visibility = ["//visibility:public"])
+
+load("@graknlabs_bazel_distribution//maven:rules.bzl", "assemble_maven", "deploy_maven")
+load("@graknlabs_dependencies//tool/checkstyle:rules.bzl", "checkstyle_test")
+load("@graknlabs_dependencies//distribution:deployment.bzl", "deployment")
+
+java_test(
+    name = "normalisation",
+    srcs = ["NormalisationTest.java"],
+    test_class = "graql.lang.pattern.test.NormalisationTest",
+    deps = [
+        # Internal Package Dependencies
+        "//java:graql",
+        "//java/query",
+        "//java/pattern",
+
+        # Internal Repository Dependencies
+
+        # External dependencies
+        "@maven//:com_google_code_findbugs_jsr305",
+    ],
+    tags = ["maven_coordinates=io.graql:graql-pattern:{pom_version}"],
```

Comment:
do we need maven coordinates here? I don't think so, right? Even if so, you shouldn't be copy-pasting the coordinates. ;)